content/getting-started/setup.textile

```diff
@@ -0,0 +1,302 @@
+
+  ```[sh]
+  pip install ably
+  pip install asyncio
+  ```
+
+  ```[python]
+  from ably import AblyRest
+  import asyncio
+  ```
+
+blang[php].
+  The PHP SDK is available as a composer package on "packagist":https://packagist.org/packages/ably/ably-php. To get started with your project, install the SDK and include the composer's autoloader. Note that the PHP SDK only implements the REST interface.
+
+  ```[sh]
+  composer require ably/ably-php --update-no-dev
+  ```
+
+  ```[php]
+  require_once __DIR__ . '/../vendor/autoload.php';
+  ```
+
+Run the following to instantiate a client:
+
+```[realtime_javascript]
+// Using promises
+const realtime = new Ably.Realtime.Promise({ key: apiKey });
+
+// Using callbacks
+const realtime = new Ably.Realtime({ key: apiKey });
+```
+
+```[realtime_nodejs]
+// Using promises
+const Ably = require('ably');
+const realtime = new Ably.Realtime.Promise({ key: apiKey });
+
+// Using callbacks
+const Ably = require('ably');
+const realtime = new Ably.Realtime({ key: apiKey });
+```
+
+```[realtime_ruby]
+EventMachine.run do
+  ably = Ably::Realtime.new(key: api_key)
+end
+```
+
+```[realtime_java]
+import io.ably.lib.types.*;
+import io.ably.lib.realtime.*;
+ClientOptions options = new ClientOptions(apiKey);
+AblyRealtime realtime = new AblyRealtime(options);
+```
+
+```[realtime_csharp]
+using IO.Ably;
+using IO.Ably.Realtime;
+ClientOptions clientOptions = new ClientOptions("<API Key>");
+AblyRealtime realtime = new AblyRealtime(clientOptions);
+```
+
+```[realtime_objc]
+ARTRealtime realtime = [[ARTRealtime alloc] initWithKey:apiKey];
+```
+
+```[realtime_swift]
+let realtime = ARTRealtime(key: apiKey)
+```
+
+```[realtime_flutter]
+final clientOptions = ably.ClientOptions(key: '<API_KEY>');
+ably.Realtime realtime = ably.Realtime(options: clientOptions);
+```
+
+```[realtime_go]
+client, err := ably.NewRealtime(ably.WithKey("{{API_KEY}}"))
+if err != nil {
+    panic(err)
+}
+```
+
+```[rest_javascript]
+var rest = new Ably.Rest({ key: apiKey });
+```
+
+```[rest_nodejs]
+var Ably = require('ably');
+var rest = new Ably.Rest({ key: apiKey });
+```
+
+```[rest_ruby]
+ably = Ably::Rest.new(key: api_key)
+```
+
+```[rest_java]
+import io.ably.lib.types.*;
+import io.ably.lib.rest.*;
+ClientOptions options = new ClientOptions(apiKey);
+AblyRest rest = new AblyRest(options);
+```
+
+```[rest_csharp]
+using IO.Ably;
+ClientOptions clientOptions = new ClientOptions(ApiKey);
+AblyRest rest = new AblyRest(clientOptions);
+```
+
+```[rest_objc]
+ARTRest rest = [[ARTRest alloc] initWithKey:apiKey];
+```
+
+```[rest_swift]
+let rest = ARTRest(key: apiKey)
+```
+
+```[rest_flutter]
+final clientOptions = ably.ClientOptions(key: '<API_KEY>');
+ably.Rest rest = ably.Rest(options: clientOptions);
+```
+
+```[rest_go]
+client, err := ably.NewREST(ably.WithKey("xxx:xxx"))
```

Comment:
All other API key representations are with some form of 'api_key', should this be the same for consistency?

Reply:
Updated and also fixed the realtime Go one.